react-app/components/FileManager/constants.js

```diff
@@ -0,0 +1,17 @@
+import { v4 as uuidv4 } from 'uuid';
+
+export const prepareCustomFileMap = () => {
+  const rootFolderId = uuidv4();
```

Comment:
I'm now generating the IDs for the files/directories. Can we remove this?

Reply:
Sorry, but you don't need to generate the ID's in the backend at this point. The idea is only to save the content tree or change the ID's if you need in that case I don't think that should be implemented when we create a new tree of content